makefiles/arch/cortexm.inc.mk

```diff
@@ -43,6 +43,32 @@ export USEMODULE += newlib
 # set default for CPU_MODEL
 export CPU_MODEL ?= $(CPU)
 
+
+# extract version inside the first parentheses
+ARM_GCC_VERSION = $(shell $(TARGET_ARCH)-gcc --version | sed -n '1 s/[^(]*(\([^\)]*\)).*/\1/p')
+
+# Ubuntu bionic gcc-arm-none-eabi compiler is not supported
+# Both when using gnu and llvm toolchains
+#
+#     /usr/bin/arm-none-eabi-gcc --version | head -n 1
+#     arm-none-eabi-gcc (15:6.3.1+svn253039-1build1) 6.3.1 20170620
+#
+ARM_GCC_UNSUPPORTED += 15:6.3.1%  # ubuntu bionic, ignore 'svn' build part
+
+# Should not raise an error on the host system version when building in docker
+ifneq (1,$(BUILD_IN_DOCKER))
+  ifneq (,$(filter $(ARM_GCC_UNSUPPORTED),$(ARM_GCC_VERSION)))
+    $(warning $(TARGET_ARCH)-gcc version not supported)
+    $(warning $(shell $(TARGET_ARCH)-gcc --version | head -n 1))
+    $(warning The currently recommanded version is the one installed in the riotdocker image)
+    $(warning https://github.com/RIOT-OS/riotdocker/blob/master/Dockerfile)
+    ifeq (1,$(WERROR))
```

Comment:
From the error message below I would expect it to be checked with `ifneq (0,$(WERROR))`

Reply:
I tried to be consistent with the other handling of WERROR

https://github.com/RIOT-OS/RIOT/blob/6a8808c5113d6ca7f5f62293a64db89393972aa1/Makefile.include#L141-L143

Reply:
Mhhhh... Ok for now, but we should think about if it isn't more clearer the other way around (afaics only `WERROR=0` is documented, nothing is said about `WERROR=<random number != 0>`